content/50.discussion.md

```diff
@@ -11,8 +11,42 @@ In this review, we seek to explain the scientific rationale underlying these tec
+However, the preponderance of the academic discourse made public is produced by western, educated, industrialized, rich and democratic, or WIERD, individuals, which comprise 12 percent of world occupants.
+While there is not much empirical data on the representativeness of WEIRD individuals within mankind, this blatant disparity brings forth the notion of bias in science, for researchers generalize their discoveries to the entire human population.
+The purpose of acknowledging the implicit biases within academia is to provide an appropriate lens for readers to interpret the information.
+
+Humans, more specifically researchers and scientists, ought to examine their ability to distinguish between evolutionary aspects of psychology and cultural aspects of psychology.
+Evolutionary theory suggests that all human psychology behaves in a similar manner.
+Thus, researchers extrapolate the idea that diversity (race, gender, and etc.) lacks importance in how they conduct their studies.
+For instance, many psychology studies utilize university college students enrolled in a psychology course for their surveys or as subjects.
+These biased results from these studies portray a false representation of global human behavior.
+An individual's environment and culture are paramount to their principles and perception, which gravely affect behavior and, consequently, scientific data.
+
+The role of bias in information and its implications goes beyond psychological studies as it is applicable to medicine: how diseases and illnesses are diagnosed, the varying degrees of treatment, and who are in clinical trials.
+The racism within medicine is critical to understanding how practitioners and researchers’ bias negatively affects health outcomes.
+There is a misconception that African-American individuals have thicker skin and exorbitant pain tolerance in comparison to other demographics.
+As a result, African-Americans can be misdiagnosed and be given incorrect prescriptions.
+The glomerular filtration rate (GFR) is a tool used to measure a specific protein in the blood, which is an indicator of kidney function.
+This sophisicalted equation inputs data, such as, weight, age, sex, serum levels of creatinine, and race, to obtain a result.
+Black individuals have a higher racial adjustment factor than that of other races.
+Thus, with similar factors as a white person, for instance, a Black person would have greater kidney function, leading to more kidney complications to reach the threshold of kidney disease.
+In addition to misdiagnosis, participation in clinical trials by African-Americans is disproportionately lower than that of caucasians, even in diseases which [African-Americans] are more susceptible.When there is equal opportunity to participate in clinical trials, there is an insignificant difference between racial groups.
+
+Why are certain racial groups disproportionately being infected with Covid-19.
```

Comment:
It sounded like this might be a section heading? Apologies if I misunderstood!
```suggestion
#### Why are certain racial groups disproportionately being infected with Covid-19
```

Reply:
both things need to be looked at: "infected" and "severely affected". These are not the same & they are both important. Don't take my word for it, take Fauci's.

Reply:
Note/reminder both things need to be looked at so worth mentioning both here in a slightly clearer way: infected and severely affected. As you know, these are not the same & they are both important, so just putting a reminder. (We had this in the section within the text & it's also said by public health officials because checking what has been missed or can be improved is one of the main strategies to stop epidemics).

Reply:
Note/reminder both things need to be looked at so worth mentioning both here in a slightly clearer way: infected and severely affected. As you know, these are not the same & they are both important, so just putting a reminder. (We had this in the section within the text & it's also said by public health officials because checking what has been missed or can be improved is one of the main strategies to stop epidemics).

Reply:
I think I understand your point. In the question I posed affected would be better than infected?